changelog_gen.py

```diff
@@ -0,0 +1,189 @@
+            e.addMeta(committer_name,committer_email)
+        else:
+            # Ideally a commit should fix a single issue, but !
+            for issue_id in issues_ids:
+                issue_details = get_issue_details(USER,PROJECT_REPO,issue_id)
+                # the issue does not correspond to any label
+                issue_labels = []
+                for label in issue_details['labels']:
+                    issue_labels.append(label['name'])
+                # a pull request always correspond to a bug fix or enchancement
+                if 'bug' in issue_labels:
+                    e = Entry('bugfix')
+                elif 'enhancement' in issue_labels:
+                    e = Entry('enhancement')
+                else:
+                    # the pulls fixes issues but they don't correspond to any labels
+                    e = Entry('fixed_issue')
+                # Set text and link
+                e.addText(issue_details['title'])
+                e.addLink(issue_details['html_url'])
+                #add author info
+                committer_name = commit.author.name
+                committer_email = commit.author.email
+                e.addMeta(committer_name,committer_email)
+
+
+# find simple commits 
+for commit in commit_list:
+    # if the commit is not checked, it can be a simple commit or it can be one that is closing an issue
+    if not (commit_lookup[commit.hex])[1]:
+        commit_message = commit.message
+        commit_lookup[commit.hex][1] = True
+        issues_ids = re.findall(r"(?<=#)\d+(?=\b)",commit_message)
+        # It is a simple commit
+        if len(issues_ids) == 0:
+            e = Entry('simple_commit')
+            e.addText(commit_message)
+        else:
+            # Ideally a commit should fix a single issue, but !
+            for issue_id in issues_ids:
+                issue_details = get_issue_details(USER,PROJECT_REPO,issue_id)
+                # the issue does not correspond to any label
+                issue_labels = []
+                for label in issue_details['labels']:
+                    issue_labels.append(label['name'])
+                # a pull request always correspond to a bug fix or enchancement
+                if 'bug' in issue_labels:
+                    e = Entry('bugfix')
+                elif 'enhancement' in issue_labels:
+                    e = Entry('enhancement')
+                else:
+                    # the pulls fixes issues but they don't correspond to any labels
+                    e = Entry('fixed_issue')
+                # Set text and link
+                e.addText(issue_details['title'])
+                e.addLink(issue_details['html_url'])
+                #add author info
+                committer_name = commit.author.name
+                committer_email = commit.author.email
+                e.addMeta(committer_name,committer_email)
+
+# print changelog
+for key in changelog_map.keys():
+    print "=> {0}".format(key.upper())
```

Comment:
As far as I know, the `print` statement is not available in Python3 (it has been replaced with the print() **function**).